src/main/java/frc/robot/commands/intake/RunIntake.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.RobotConfig;
+import frc.robot.subsystems.intake.Intake;
+
+public class RunIntake extends Command {
+  private final Intake m_intake;
+
+  /**
+   * Creates a new RunIntake command.
+   *
+   * @param subsystem The subsystem used by this command.
+   */
+  public RunIntake(Intake subsystem) {
+    m_intake = subsystem;
+
+    addRequirements(subsystem);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
```

Comment:
intake should be run in periodic

Reply:
Moved to execute in "moved run to execute" commit